metadata-io/src/main/java/com/linkedin/metadata/recommendation/candidatesource/MostPopularSource.java

```diff
@@ -106,7 +110,7 @@ private SearchRequest buildSearchRequest(@Nonnull Urn userUrn) {
     // Find the entities with the most views
     AggregationBuilder aggregation = AggregationBuilders.terms(ENTITY_AGG_NAME)
         .field(DataHubUsageEventConstants.ENTITY_URN + ESUtils.KEYWORD_SUFFIX)
-        .size(MAX_CONTENT);
+        .size(MAX_CONTENT * 2);
```

Comment:
Curious - why this change necessary?

Reply:
so that after filtering we still have the num left